float32 x_state
float32 y_state
bool ready
int32 agent_id
float32 attrition_estimate
int32 target_id
float32 effectiveness

Model[] models
